### PR TITLE
Logging memory usage

### PR DESCRIPTION
**Pull request by bashkarev**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  | https://github.com/yiisoft/yii2-debug/issues/34, https://github.com/yiisoft/yii2-debug/pull/184,

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by SilverFire — APPROVED** [review]

File: framework/log/Logger.php [hunk 1]
```diff
@@ -263,27 +264,29 @@ public function getDbProfiling()
...
     public function calculateTimings($messages)
     {
         $timings = [];
         $stack = [];
 
         foreach ($messages as $i => $log) {
-            list($token, $level, $category, $timestamp, $traces) = $log;
-            $log[5] = $i;
+            list($token, $level, $category, $timestamp, $traces, $memory) = $log;
+            $log[6] = $i;
```
Comment (dynasource) [inline framework/log/Logger.php]: would be nice to fix this hardcoded number

**Review by dynasource — APPROVED** [review]

File: framework/log/Logger.php (hunk 1 above)
Comment (samdark) [inline framework/log/Logger.php]: I don't think it can be done without compatibility breaks.

**Comment by SilverFire** [comment]
Could be merged in 2.0.11 :christmas_tree:

**Comment by samdark** [comment]
Need a line for changelog.

**Review by cebe — APPROVED** [review]

File: framework/log/Logger.php [hunk 2]
```diff
@@ -263,27 +264,29 @@ public function getDbProfiling()
...
      */
     public function calculateTimings($messages)
     {
         $timings = [];
         $stack = [];
 
         foreach ($messages as $i => $log) {
-            list($token, $level, $category, $timestamp, $traces) = $log;
-            $log[5] = $i;
+            list($token, $level, $category, $timestamp, $traces, $memory) = $log;
```
Comment (cebe) [inline framework/log/Logger.php]: this will break on old logging data, when memory is not set. Will fix that while merging.

**Pull request closed without merging**